ci/android.groovy

```diff
@@ -28,7 +28,7 @@ def uploadToSauceLabs() {
   if (changeId != null) {
     env.SAUCE_LABS_NAME = "${changeId}.apk"
   } else {
-    env.SAUCE_LABS_NAME = "im.status.ethereum-e2e-${GIT_COMMIT.take(6)}.apk"
+    env.SAUCE_LABS_NAME = "${pkg}"
```

Comment:
Where does `pkg` come from in that context? I don't see it's source.

Reply:
from https://github.com/status-im/status-react/pull/7279/files/8eedc83cc57ef5ffd826dc3abab5c5a6b07655e4#diff-921ddb8a284f72367049b92eff26b9b6R12
Or I can't use this in uploadToSauceLabs?

Reply:
Nope, it's a context of a different method, so I doubt that will work.

Reply:
It seems that I should call `cmn.pkgFilename(type, 'apk')`, right?

Reply:
Yeah, that should work. I don't think passing that from the original place would make for nice code, that should be just fine.

Reply:
i've added the definition of pkg to the method